xlators/mgmt/glusterd/src/glusterd-volgen.c

```diff
@@ -118,7 +118,7 @@ xlator_instantiate_va(const char *type, const char *format, va_list arg)
     xl->name = volname;
     CDS_INIT_LIST_HEAD(&xl->volume_options);
 
-    xl->ctx = this->ctx;
+    xl->ctx = global_ctx;
```

Comment:
Do we really need this ? I think we should remove the `ctx` field from `xlator_t`.